image-filters/image-filters.markdown

```diff
@@ -1552,16 +1551,18 @@ Making things look just so. Making them fail over. Maintaining that 99.9%
 uptime. We spend more time hunting down corner cases than refining algorithms.
```

Comment:
If it's not your text I would remove it --- it doesn't really make sense in the context of this chapter anyway; it's not like you're going to failover test an image filter app...  Can you think of a more appropriate thing to add to the list of "stuff we do in productionisation"? Maybe "Testing edge cases."?